demos/_unit-test/scope-builder.php

```diff
@@ -15,20 +17,33 @@
 // Display: Project Name matches regular expression [a-zA-Z]
 $project = new Condition('project_name', Condition::OPERATOR_REGEXP, '[a-zA-Z]');
 // Display: Client Country Iso equals Brazil => ToWord: Client Country Iso is equal to 'Brazil'
-$brazil = new Condition('client_country_iso', '=', 'Brazil');
+$brazil = new Condition('client_country_iso', Condition::OPERATOR_EQUALS, 'Brazil');
```

Comment:
well, I think `=` was invented for humans and well standardized, @georgehristov wdyt? Is there even any sense making this constant public?

Reply:
What is the status here? should we use condition const or symbol?

Reply:
We should use symbols I would say

Reply:
thanks, I will change it back to symbol.